docs/docs-content/clusters/public-cloud/aws/add-aws-accounts.md

```diff
@@ -40,95 +40,101 @@ To add an AWS cloud account using static access credentials follow these steps:
+8. Once the credentials are validated, the **Add IAM Policies** toggle displays. Toggle **Add IAM Policies** on.
 
-8. A drop-down menu displays a lists of available AWS IAM policies in your AWS account. Select any desired IAM policies you want to assign to Palette IAM role or IAM user.
+9. Use the **drop-down Menu**, which lists available IAM policies in your AWS account, to select any desired IAM policies you want to assign to Palette IAM role or IAM user.
 
 
 #### Validate
 
-You can validate the account is available in Palette by reviewing the list of cloud accounts. To review the list of cloud accounts navigate to the left **Main Menu**. Click on **Tenant Settings**. Next, click on **Cloud Accounts**. Your newly added AWS cloud account is listed under the AWS sections.
+You can validate the account is available in Palette by reviewing the list of cloud accounts. To review the list of cloud accounts, navigate to the left **Main Menu** and click on **Tenant Settings**. Next, click on **Cloud Accounts**. Your newly added AWS cloud account is listed under the AWS section.
 
 
 
 ### Dynamic Access Credentials
 
-To add an AWS cloud account using STS credentials follow the steps below:
+Use the steps below to add an AWS cloud account using Security token service (STS) credentials.
 
 #### Prerequisites
 
-- An AWS account
+- An AWS account.
 - Sufficient access to create an IAM role or IAM user.
-- Palette IAM policies. Please review the [Required IAM Policies](required-iam-policies.md) section for guidance.
+- Palette IAM policies. Review the [Required IAM Policies](required-iam-policies.md) section for guidance.
 
 
 #### Add AWS Account to Palette
 
-1. Log in to [Palette](https://console.spectrocloud.com) as Tenant admin.
+1. Log in to [Palette](https://console.spectrocloud.com) as tenant admin.
 
 
-2. Go to **Tenant Settings** > **Cloud Accounts** and click **+Add AWS Account**.
+2. From the left **Main Menu**, click on **Tenant Settings**.
 
 
-3. In the cloud account creation wizard give the following information:
-   * **Account Name**
-   * **Description**
-   * Select **STS** authentication for validation:
+3. Select **Cloud Accounts**, and click **+Add AWS Account**.
+
+
+4. In the cloud account creation wizard give the following information:
+   * **Account Name**:  Custom name for the cloud account.
+   * **Description**: Optional description for the cloud account.
+   * Select **STS** authentication for validation.
 
 
-4. You will be provided with information on the right hand-side of the wizard. You will need this information to create an IAM Role for Palette. The following table lists out the information provided by the wizard after your selects **STS**.
+5. You will be provided with information on the right side of the wizard. You will need this information to create an IAM Role for Palette. The following table lists the information provided by the wizard after your select **STS**.
```

Comment:
there is a spelling mistake here "after your select"

Reply:
fixed.